homework.md

```diff
@@ -44,61 +44,64 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language used to make a website interactive and give it functionality.
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+It allows people to share their code and collaborate. 
+It gives a timeline of changes
+We can revert back to an earlier version if we need to change code. 
+
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+Make a remote copy of a repository. 
 
 ## Question 10 
 
 What happens when you clone a repostory?
 
-It send it to a friend
+A local copy of a repository 
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+It allows you to show others changes you have pushed to a branch in a repo on GitHub.
 
 ## Question 12
 
 What is the Git History log?
 
-It lets you see into the future
+A list of commits made in reverse chronological order. 
 
 ## Question 13
 
 What does it meant to "push" changes to Github?
 
-You send the changes of your local repository to Github
+Push local changes to the remote repo 
 
 ## Question 14
 
 What is Github?
 
-A place for us to store and share our code.
+Version control software. 
```

Comment:
Hi git is version control software. github is where we share and work together on the project online.

Reply:
Thank you for the feedback, I will change it